frontend/compositions/dashboardHeader/index.tsx

```diff
@@ -15,6 +15,7 @@ export default function DashboardHeader() {
     desktopTitle,
     rightHeader,
     nav,
+    buttonContainer,
     mobileNav,
     desktopNav,
```

Comment:
Non-blocker but this list is getting a little lengthy - I find utilizing nesting can help reduce the number of styles names you have to import/deconstruct if that helps pare it down some!

Reply:
@srsexton94 Thank you for the feedback and the tip on nesting- definitely agree that this list is getting lengthy/there are too many unnecessary divs. I went back through to nest redundant class names, and was able to cut six style names from this section.